include/hdf5_particle.h

```diff
@@ -39,8 +38,33 @@ typedef struct HDF5Particle {
   mpm::Index cell_id;
   // Status
   bool status;
+  // Material id
+  unsigned material_id;
+  // Number of state variables
+  unsigned nstate_vars;
+  // State variables (init to zero)
+  double svars[20] = {0};
```

Comment:
Will you print all the svars value, even if it's not initialized, i.e. equal to 0, to the hdf5?

Reply:
That's the reason to have a fixed number of svars, otherwise we may as well have it as a vector

Reply:
It's fine to store them as a vector with a fix size. I mean when you print, are we printing all the vector values even it's 0? Or is there any check that we have to adaptively print depends on `nstate_vars`?

Reply:
You mean, when we write to HDF5, yes, it's a table with a fixed size. HDF5 table is our restriction to have a fixed size, not the struct itself.